contribs/commercialTrafficApplications/src/main/java/org/matsim/contrib/commercialTrafficApplications/jointDemand/commercialJob/DefaultCommercialJobGenerator.java

```diff
@@ -66,39 +68,43 @@
  */
 class DefaultCommercialJobGenerator implements CommercialJobGenerator {
 
-
 	private final double firsttourTraveltimeBuffer;
-    private final int timeSliceWidth;
-
-    private Scenario scenario;
-
-    private Carriers carriers;
-
-    private final TravelTime carTT;
-
-    private Set<Id<Person>> freightDrivers = new HashSet<>();
-    private Set<Id<Vehicle>> freightVehicles = new HashSet<>();
-
-    private final static Logger log = Logger.getLogger(DefaultCommercialJobGenerator.class);
-
-    private Set<String> drtModes = new HashSet<>();
-
-    @Inject
-    /* package */ DefaultCommercialJobGenerator(Scenario scenario, Map<String, TravelTime> travelTimes, Carriers carriers ) {
-        JointDemandConfigGroup cfg = JointDemandConfigGroup.get(scenario.getConfig());
-        this.carriers = carriers;
-        this.firsttourTraveltimeBuffer = cfg.getFirstLegTraveltimeBufferFactor();
-        this.timeSliceWidth = ConfigUtils.addOrGetModule(scenario.getConfig(), FreightConfigGroup.class).getTravelTimeSliceWidth();
-        this.scenario = scenario;
-        carTT = travelTimes.get(TransportMode.car);
-        getDrtModes(scenario.getConfig());
-    }
+	private final int timeSliceWidth;
+	private StrategyManager strategyManager;
+	private Scenario scenario;
+	private int keepUsedOperatorsInterval;
```

Comment:
name the variable changeCommercialJobOperatorInterval?
Or changeJobOperatorInterval?
Or changeOperatorInterval?

Reply:
Switch it to changeOperatorInterval